packages/storefront/src/components/CodeBlock.vue

```diff
@@ -103,6 +103,7 @@
           .token.symbol,
           .token.deleted {
             color: #007299;
+            font-family: Monaco, Menlo, 'Andale Mono', 'Ubuntu Mono', monospace;
```

Comment:
I think this could go some levels higher, specified once and then inherited here and in the other place.